openedx/core/djangoapps/coursegraph/management/commands/dump_to_neo4j.py

```diff
@@ -188,56 +191,62 @@ def get_command_last_run(course_key, graph):
     def get_course_last_published(course_key):
         """
         We use the CourseStructure table to get when this course was last
-          published.
-        :param course_key: a CourseKey
-        :return: a string representation of when this course was last dumped to
-          neo4j or None
+        published.
+        Args:
+            course_key: a CourseKey
+
+        Returns: None, if there's no record of the last time this course was
```

Comment:
similar to my last nit. Or at least, if you take my nit, you should apply it universally!